src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -1486,6 +1486,13 @@ namespace {
                 auto gas_target = record.getItem("GAS_TARGET").get<UDAValue>(0);
                 auto water_target = record.getItem("WATER_TARGET").get<UDAValue>(0);
                 auto liquid_target = record.getItem("LIQUID_TARGET").get<UDAValue>(0);
+                double guide_rate = -1;
+                std::string guide_rate_def;
+                if (record.getItem("GUIDE_RATE_DEF").hasValue(0)) {                  
+                  guide_rate_def = record.getItem("GUIDE_RATE_DEF").get<std::string>(0);
+                  if (! (guide_rate_def == "INJ" || guide_rate_def == "POTN" || guide_rate_def == "FORM"))
+                    guide_rate = record.getItem("GUIDE_RATE").get<double>(0);
```

Comment:
There could be a check: if a value has been assigned to the guide_rate and the the guide_rate_def is in the set `["INJ", "POTN", "FORM"]`:
```
if (!record.getItem("GUIDE_RATE").defaultApplied(0))
    if (guide_rate_def == "INJ" || guide_rate_def == "POTN" || .. ) {
         std::string msg = "The supplied guide_rate value will be ignored";
         parseContext.handleError(ParseContext::SCHEDULE_IGNORED_GUIDE_RATE, errors, msg);
   }
}
```
Before this will actually work you must look a bit at the `ParseContext` machinery - which is used to configure/control how severely errors should be treated; and a new `ParseContext` e.g. `SCHEDULE_IGNORED_GUIDE_RATE` to control the behavior for this particular error.